packages/cozy-stack-client/src/__tests__/DocumentCollection.spec.js

```diff
@@ -243,6 +243,22 @@ describe('DocumentCollection', () => {
       )
     })
 
+    it('should only use one sort order', async () => {
+      const collection = new DocumentCollection('io.cozy.todos', client)
+      await collection.find({ done: false }, { sort: { label: 'asc', _id: 'desc' } })
+      expect(client.fetchJSON).toHaveBeenLastCalledWith(
+        'POST',
+        '/data/io.cozy.todos/_find',
+        {
+          limit: 50,
+          skip: 0,
+          selector: { done: false },
+          sort: [{ _id: 'asc' }, { done: 'asc' }, { label: 'asc' }],
```

Comment:
IMHO, this is very surprising from the point of view of the user. They tell the library to get results in a certain order, and the library does something behind her back to change that ?